File: app/src/main/java/org/take365/Network/Models/StoryDetailsModel.java
package org.take365.Network.Models;

import java.util.List;

/**
 * Created by Ermakov-MAC on 18.02.16.
 */
public class StoryDetailsModel {
    public List<AuthorModel> authors;
    public int id;
    public List<StoryImageImagesModel> images;
    public StoryProgressModel progress;
    public int status;
    public String title;
    public String url;
}

File: app/src/main/java/org/take365/Network/Models/StoryDetailsModel.kt
package org.take365.Network.Models

/**
 * Created by Ermakov-MAC on 18.02.16.
 */
data class StoryDetailsModel(
        var authors: List<AuthorModel>,
        var id: Int,
        var images: List<StoryImageImagesModel>,
        var progress: StoryProgressModel,
        var status: Int,
        var title: String?,
        var url: String?
)
